classes/mender-install.bbclass

```diff
@@ -1,6 +1,10 @@
 PREFERRED_VERSION_go_cross = "1.6%"
 
-#Add support for systemd
+# Add support for systemd
+#
+# Note that a very common error with systemd is that the system hangs during the
+# boot process while looking for devices. This is almost always because the
+# kernel feature CONFIG_FHANDLE is not enabled.
 DISTRO_FEATURES_append = " systemd"
```

Comment:
Just document that systemd is needed then, but neven enable it in some random place.

Also update mender recipe to ship mender.service only if systemd is enabled in DISTRO_FEATURES. You can check like this:
```
if ${@bb.utils.contains('DISTRO_FEATURES', 'systemd', 'true', 'false', d)}; then
    install -t ${systemd_unitdir}system/..
fi
```